src/harness/fake_sas.py

```diff
@@ -222,6 +237,18 @@ def do_POST(self):
     self.end_headers()
     self.wfile.write(json.dumps(response))
 
+  def do_GET(self):
+    """Handles GET requests."""
+    path, value = self._parseUrl(self.path)
+    if path == "/v1.0/sas_impl":
```

Comment:
The URL path is basically the same for SAS-CBSD, right? So the approach which works for SAS-CBSD should also work for SAS-SAS, I think... Am I missing something?